### DISCUSSION TITLE
q-table grouping headers

### DISCUSSION THREAD

**Discussion by zanzara** [quasarframework/quasar#8768] [General - Components / Directives / etc]
Is it somehow possible to group two header columns?
i.e.: 

```
      |    timestamp      |          |          
col1  |  from  | until    |   col4   | etc.
```

**Comment by metalsadman** [comment]
use header slot, and since q-table just wraps html table then you can use colspan on your q-th's/q-td's. rough example https://codepen.io/metalsadman/pen/ZELyXVq?editors=1010.

**Reply by zanzara** [reply]
Yes, thanks. Good approach. That might help to get it fix.
But have to tinker still a bit, cause user has the possibility to make cols visible or not.

**Reply by iamjuanguillermo** [reply]
Hi! Were you able to find the solution?
